### PR TITLE
Remove the Documentation folder from this repo

### PR DESCRIPTION
**Pull request by MaxWang-MS**
## Overview
As the docs have been migrated to DMC, deleting the documentation folder in this repo.

### CODE REVIEW

File: Documentation/Authors.md [hunk 1]
```diff
@@ -1,133 +0,0 @@
-# Authors
```
Comment (keveleigh) [inline Documentation/Authors.md:1]: Can we move this file to the repo root? I'm not sure it was migrated to DMC and I'd like to keep this list around!

Comment (MaxWang-MS) [reply Documentation/Authors.md:1]: Sure!

**Review by keveleigh — APPROVED** [review]

**Comment by davidkline-ms** [comment]
@MaxWang-MS, the mrtk_docs pipeline will need to be disabled for this to pass :)

**Review by davidkline-ms — APPROVED** [review]

**Comment by MaxWang-MS** [comment]
> @MaxWang-MS, the mrtk_docs pipeline will need to be disabled for this to pass :)

Hi @davidkline-ms, I actually think the CI is reporting some valid concerns... As I just mentioned offline there are some references to images in the deleted Documentation folder in some markdown files in the Assets folder. I will be removing those markdown files and try running the CI again.

**Review by CDiaz-MS — APPROVED** [review]

**Pull request merged**